base/tests/unittests/com/google/idea/blaze/base/sync/workspace/ArtifactLocationDecoderTest.java

```diff
@@ -57,66 +57,6 @@ public void testGeneratedArtifact() {
         .isEqualTo(EXECUTION_ROOT + "/blaze-out/bin/com/google/Bla.java");
   }
 
-  @Test
-  public void testExternalSourceArtifactOldFormat() {
-    ArtifactLocation artifactLocation =
-        ArtifactLocation.fromProto(
-            Common.ArtifactLocation.newBuilder()
-                .setRelativePath("external/repo_name/com/google/Bla.java")
-                .setIsSource(true)
-                .setIsExternal(true)
-                .build());
-
-    assertThat(artifactLocation.getRelativePath()).isEqualTo("com/google/Bla.java");
-    assertThat(artifactLocation.getExecutionRootRelativePath())
-        .isEqualTo("external/repo_name/com/google/Bla.java");
-
-    ArtifactLocationDecoder decoder =
-        new ArtifactLocationDecoderImpl(
-            BlazeInfo.createMockBlazeInfo(
-                OUTPUT_BASE,
-                EXECUTION_ROOT,
-                EXECUTION_ROOT + "/blaze-out/crosstool/bin",
-                EXECUTION_ROOT + "/blaze-out/crosstool/genfiles",
-                EXECUTION_ROOT + "/blaze-out/crosstool/testlogs"),
-            null,
-            RemoteOutputArtifacts.EMPTY);
-
-    assertThat(decoder.decode(artifactLocation).getPath())
-        .isEqualTo(EXECUTION_ROOT + "/external/repo_name/com/google/Bla.java");
-  }
-
-  @Test
-  public void testExternalDerivedArtifactOldFormat() {
-    ArtifactLocation artifactLocation =
-        ArtifactLocation.fromProto(
-            Common.ArtifactLocation.newBuilder()
-                .setRelativePath("external/repo_name/com/google/Bla.java")
-                .setRootExecutionPathFragment("blaze-out/crosstool/bin")
-                .setIsSource(false)
-                .setIsExternal(true)
-                .build());
-
-    assertThat(artifactLocation.getRelativePath()).isEqualTo("com/google/Bla.java");
-    assertThat(artifactLocation.getExecutionRootRelativePath())
-        .isEqualTo("blaze-out/crosstool/bin/external/repo_name/com/google/Bla.java");
-
-    ArtifactLocationDecoder decoder =
-        new ArtifactLocationDecoderImpl(
-            BlazeInfo.createMockBlazeInfo(
-                OUTPUT_BASE,
-                EXECUTION_ROOT,
-                EXECUTION_ROOT + "/blaze-out/crosstool/bin",
-                EXECUTION_ROOT + "/blaze-out/crosstool/genfiles",
-                EXECUTION_ROOT + "/blaze-out/crosstool/testlogs"),
-            null,
-            RemoteOutputArtifacts.EMPTY);
-
-    assertThat(decoder.decode(artifactLocation).getPath())
-        .isEqualTo(
-            EXECUTION_ROOT + "/blaze-out/crosstool/bin/external/repo_name/com/google/Bla.java");
-  }
-
   @Test
   public void testExternalSourceArtifactNewFormat() {
```

Comment:
nit: can you remove `NewFormat` from the test name here and in other test methods?